### PR TITLE
add empathy irc regression test

### PR DESCRIPTION
**Pull request by mitiao**
I am only adding this empathy irc test to Regression, and other empathy regression tests are written by Dehai who will submit them to upstream.

The merge request of needles for empathy irc was submitted on gitlab as well.

### CODE REVIEW

File: tests/x11regressions/empathy/empathy_irc.pm [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
```
Comment (coolo) [inline tests/x11regressions/empathy/empathy_irc.pm]: please only add 2016 as copyright. Bernhard has no work done here

Comment (mitiao) [inline tests/x11regressions/empathy/empathy_irc.pm]: removed the line of Bernhard.

**Comment by coolo** [comment]
the correct line is just 2016 SUSE LLC

but I don't want to stop the PR for that, but please keep that in mind for future tests

**Comment by mitiao** [comment]
ah, I see.
I will keep it in my mind and fix it in next regression commit

**Pull request merged**